src/System.Security.Permissions/src/System/Security/Permissions/MediaPermission.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+namespace System.Security.Permissions
+{
+    public enum MediaPermissionAudio
+    {
+        NoAudio,
+        SiteOfOriginAudio,
+        SafeAudio,
+        AllAudio
+    }
+    public enum MediaPermissionVideo
+    {
+        NoVideo,
+        SiteOfOriginVideo,
+        SafeVideo,
+        AllVideo,
+    }
+    public enum MediaPermissionImage
+    {
+        NoImage,
+        SiteOfOriginImage,
+        SafeImage,
+        AllImage,
+    }
+    [Serializable()]
```

Comment:
Do you have a real scenario for making these Serializable?  We removed [Serializable] on all other permission types, only keeping it on exceptions.  In fact the base type of these is no longer Serializable so you this won't even work.  I'd recommend removing Serializable attribute and TypeForwardedFrom.

Reply:
No objections to removing these.